deb-get

```diff
@@ -2413,6 +2413,20 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_xournalpp() {
+    CODENAMES_SUPPORTED="bionic bullseye focal"
+    get_github_releases "https://api.github.com/repos/xournalpp/xournalpp/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        if [[ "${UPSTREAM_CODENAME}" == *"${CODENAMES_SUPPORTED}"* ]]; then
+            URL="$(grep "browser_download_url.*.deb" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)"
```

Comment:
To `grep` the correct URL, you could do something like `"browser_download_url.*${UPSTREAM_CODENAME}-x86_64\.deb\""`.

Reply:
thank you for all the suggestions on how to improve my code additions ... I addressed a few of these issues (pulled down some PR's for now, since they clearly aren't ready from my end, i realize that now) 

this issue i have hopefully addressed but i need to get a bullseye or focal VM loaded to test it ... was wondering why my latest test code wasn't returning a URL but i am on a Jammy base and this app doesn't support Jammy (last I checked)